.azure-pipelines/release-cli.yaml

```diff
@@ -0,0 +1,406 @@
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - refs/tags/v*
+  paths:
+    include:
+    - .azure-pipelines
+    - src
+    - msgraph-cli.sln
+
+pool:
+  name: Azure Pipelines
+  vmImage: ubuntu-latest
+
+variables:
+  - name: repositoryConnection
+    value: 'GitHub - calebkiage'
```

Comment:
Can you use another connection that's not specific to a user to avoid connections duplication?

Reply:
Do we already have a connection that can be reused? I can try it.

Reply:
recap from our discussion on team, there's no main connection, we can leave it as is for now